client/src/actions/scrape-fcc.js

```diff
@@ -29,7 +29,7 @@ export const scrapeFccStats = (user) => (dispatch) => {
         return item.text();
       });
     }).then(items => {
-      return items.length;
+      return items.length.toString();
```

Comment:
☝️ whoops, this was not necessary, was messing around because total challenges was not rendering, but this was apparently because of a long time typo that went unnoticed (`totalChallneges`)